client/src/icons/Logo.js

```diff
@@ -1,58 +1,78 @@
 import React from 'react';
 import styled from '@emotion/styled';
 import { fadeIn } from '../utils/animations';
+import logo from '../icons/logo.svg';
```

Comment:
`logo.svg` contains the fork. I would say that the logo is more than the fork and contains the text too.
I wouldn't split the logo into multiple files.